redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+        if msg:
+            for page in pagify(msg):
+                await ctx.maybe_send_embed(page)
+            return
+        await ctx.maybe_send_embed(_("There are no mutes on this server right now."))
+
+    @commands.command(usage="[users...] [time_and_reason]")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def mute(
+        self,
+        ctx: commands.Context,
+        users: commands.Greedy[discord.Member],
+        *,
+        time_and_reason: MuteTime = {},
+    ):
+        """Mute users.
+
+        `[users...]` is a space separated list of usernames, ID's, or mentions.
+        `[time_and_reason]` is the time to mute for and reason. Time is
+        any valid time length such as `30 minutes` or `2 days`. If nothing
+        is provided the mute will be indefinite.
+
+        Examples:
+        `[p]mute @member1 @member2 spam 5 hours`
+        `[p]mute @member1 3 days`
+
+        """
+        if not users:
+            return await ctx.send_help()
+        if ctx.me in users:
+            return await ctx.send(_("You cannot mute me."))
+        if ctx.author in users:
+            return await ctx.send(_("You cannot mute yourself."))
+        if not await self._check_for_mute_role(ctx):
+            return
+        async with ctx.typing():
+            duration = time_and_reason.get("duration", {})
+            reason = time_and_reason.get("reason", None)
+            time = ""
+            until = None
+            if duration:
+                until = datetime.now(timezone.utc) + timedelta(**duration)
+                time = _(" for ") + humanize_timedelta(timedelta=timedelta(**duration))
+            else:
+                default_duration = await self.config.guild(ctx.guild).default_time()
+                if default_duration:
+                    until = datetime.now(timezone.utc) + timedelta(**default_duration)
+                    time = _(" for ") + humanize_timedelta(timedelta=timedelta(**default_duration))
+            author = ctx.message.author
+            guild = ctx.guild
+            audit_reason = get_audit_reason(author, reason)
+            success_list = []
+            for user in users:
+                success, issue = await self.mute_user(guild, author, user, until, audit_reason)
+                if success:
+                    success_list.append(user)
+                    try:
+                        await modlog.create_case(
+                            self.bot,
+                            guild,
+                            ctx.message.created_at.replace(tzinfo=timezone.utc),
+                            "smute",
+                            user,
+                            author,
+                            reason,
+                            until=until,
+                            channel=None,
+                        )
+                    except RuntimeError as e:
+                        log.error(_("Error creating modlog case"), exc_info=e)
```

Comment:
Logging should not be translated.
```suggestion
                        log.error("Error creating modlog case", exc_info=e)
```